node/http/src/api.rs

```diff
@@ -0,0 +1,186 @@
+            receiver: r.receiver_account_id,
+            amount: r.amount,
+        });
+        debug!(target: "near-rpc", "Send money transaction {:?}->{:?}, amount: {:?}",
+               r.sender_account_id, r.receiver_account_id, r.amount);
+        Ok(PreparedTransactionBodyResponse { body })
+    }
+
+    pub fn stake(
+        &self,
+        r: &StakeRequest,
+    ) -> Result<PreparedTransactionBodyResponse, ()> {
+        let body = TransactionBody::Stake(StakeTransaction {
+            nonce: r.nonce,
+            staker: r.staker_account_id,
+            amount: r.amount,
+        });
+        debug!(target: "near-rpc", "Stake money transaction {:?}, amount: {:?}",
+               r.staker_account_id, r.amount);
+        Ok(PreparedTransactionBodyResponse { body })
+    }
+
+    pub fn schedule_function_call(
+        &self,
+        r: ScheduleFunctionCallRequest,
+    ) -> Result<PreparedTransactionBodyResponse, ()> {
+        debug!(target: "near-rpc", "Schedule function call transaction {:?}.{:?}",
+               r.contract_account_id, r.method_name);
+        let body = TransactionBody::FunctionCall(FunctionCallTransaction {
+            nonce: r.nonce,
+            originator: r.originator_account_id,
+            contract_id: r.contract_account_id,
+            method_name: r.method_name.into_bytes(),
+            args: r.args,
+            amount: r.amount,
+        });
+        Ok(PreparedTransactionBodyResponse { body })
+    }
+
+    pub fn view_account(
+        &self,
+        r: &ViewAccountRequest,
+    ) -> Result<ViewAccountResponse, String> {
+        debug!(target: "near-rpc", "View account {:?}", r.account_id);
+        match self.state_db_viewer.view_account(r.account_id) {
+            Ok(r) => {
+                Ok(ViewAccountResponse {
+                    account_id: r.account,
+                    amount: r.amount,
+                    stake: r.stake,
+                    code_hash: r.code_hash,
+                    nonce: r.nonce,
+                })
+            }
+            Err(e) => { Err(e.to_string()) }
+        }
+    }
+
+    pub fn call_view_function(
+        &self,
+        r: &CallViewFunctionRequest,
+    ) -> Result<CallViewFunctionResponse, String> {
+        debug!(
+            target: "near-rpc",
+            "Call view function {:?}{:?}",
+            r.contract_account_id,
+            r.method_name,
+        );
+        match self.state_db_viewer.call_function(
+            r.originator_id,
+            r.contract_account_id,
+            &r.method_name,
+            &r.args,
+        ) {
+            Ok(result) => {
+                Ok(CallViewFunctionResponse { result })
+            }
+            Err(e) => { Err(e.to_string()) }
+        }
+    }
+
+    pub fn submit_transaction(&self, r: SignedTransaction) -> Result<(), ()> {
+        debug!(target: "near-rpc", "Received transactio\
+        n {:?}", r);
+        self.submit_txn_sender.clone().try_send(r).unwrap();
```

Comment:
This will fail if the channel is full. Should return some meaningful error message if that happens instead of panicking.